.github/workflows/nightly.yml

```diff
@@ -0,0 +1,41 @@
+name: "Nightly Build"
+on:
+  schedule:
+    - cron: "10 * * * *"
```

Comment:
As configured, this will run at 10 past every hour (that is not the long term plan, just to make testing easier)